utils/epinow2/functions.py

```diff
@@ -0,0 +1,122 @@
+from uuid import uuid1
+from datetime import datetime, timedelta
+from utils.epinow2.constants import states, pathogens, nssp_states_omit, shared_params
+
+
+def generate_job_id():
+    """Generates a UUID1 object to associate
+    with job IDs. UUID1 can be sorted by
+    timestamp by default, which is desirable
+    when managing multiple configurations."""
+    return uuid1()
```

Comment:
Using UUID1 as it seems time-sortable.

Reply:
It looks like there is a chance that UUID1 will [not always](https://stackoverflow.com/questions/8713873/is-python-uuid1-sequential-as-timestamps) be time sortable. That said, I think it's fine to use it, as long as we don't 100% rely on it being time sortable.